eng/CHECKENFORCER

```diff
@@ -55,4 +55,6 @@ message: >
   
   ```
   /azp run net - [service] - ci
-  ```
\ No newline at end of file
+  ```
+  
+  **Note: For new Microsoft.Azure.Management.\* SDKs, please apply `new service` label so reviewer can help set up the pipeline bootstrap.**
```

Comment:
Should we instead provide the longer description of the steps to update the mgmt.yml file?

Reply:
That's a good idea. I will check internally to see if there are any objections.

Reply:
Yeah, I generally have them do this, it is pretty easy to understand